views/js/review/plugins/navigation/review-panel/panel-data.js

```diff
@@ -60,16 +61,17 @@ define([
         if (item.informational) {
             return 'info';
         }
+        //if item was never visited, item.score=null. So it will be 'incorrect', not 'skipped'. Is it intentional?
+        if (item.skipped || item.score === null) {
+            return 'skipped';
+        }
```

Comment:
The mentioned "bugfix" which could be controversial.

Reply:
Will be reverted soon.

Reply:
About informational:
- notice that for **`item-answer` part in the right side** of the page, not-visited items have incorrect score now. Before they had no score as they were 'informational'. Let's be careful with that.
- if we want to keep informational detection change, maybe show not-visited items with _thin_ border?

Reply:
I've checked this last point with BA. It's ok to show the `0/0` score on these items. It should be a quite rare case anyway.

Reply:
Ok then.